components/movementsensor/gpsutils/vrs.go

```diff
@@ -190,8 +204,8 @@ func (vrs *VRS) ReadLine() (string, error) {
 	return string(line), nil
 }
 
-// WriteLine writes a line to the vrs's readerWriter.
-func (vrs *VRS) WriteLine(line string) error {
+// WriteString writes a string to the vrs's readerWriter.
+func (vrs *VRS) WriteString(line string) error {
```

Comment:
This isn't technically writing a line (e.g. it didn't provide line endings) so renamed it.